### PR TITLE
Fix multiple CS errors

### PR DESCRIPTION
**Pull request by alexandre-daubois**
| Q             | A
| ------------- | ---
| Branch?       | 7.2
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | -
| License       | MIT

My first intention was to add a missing backslash to calls to `sprintf` as @derrabus did a few months ago, before thinking that perhaps a CS Fixer pass might be beneficial. Here are some notable improvements thanks to this pass:

- The addition of backslashes for `sprintf`, even more important since the improvements to the PHP engine
- Addition of trailing commas in array declarations, but especially when using CPP, as recently "standardized" in a recent PR
- Reordering (or better, removal) of a few (sometimes uesless) `use` statements
- Some `array_key_exists` were missing a leading backslash, preventing the engine to use the opcode dedicated to it
- Added leading backslash to a few calls PHP CS Fixer assumed beneficial

### CODE REVIEW

File: src/Symfony/Bridge/PhpUnit/ClockMock.php [hunk 1]
```diff
@@ -100,12 +100,12 @@ public static function hrtime($asNumber = false)
     {
         $ns = (self::$now - (int) self::$now) * 1000000000;
 
+
```
Comment (derrabus) [inline src/Symfony/Bridge/PhpUnit/ClockMock.php]: That additional line does not look right.

File: src/Symfony/Bridge/PhpUnit/ClockMock.php [hunk 2]
```diff
@@ -100,12 +100,12 @@ public static function hrtime($asNumber = false)
...
         if ($asNumber) {
-            $number = sprintf('%d%d', (int) self::$now, $ns);
+            $number = \sprintf('%d%d', (int) self::$now, $ns);
 
             return \PHP_INT_SIZE === 8 ? (int) $number : (float) $number;
         }
-
```
Comment (derrabus) [inline src/Symfony/Bridge/PhpUnit/ClockMock.php]: … neither does this line removal.

File: src/Symfony/Bridge/PhpUnit/ClockMock.php (hunk 1 above)
Comment (alexandre-daubois) [reply src/Symfony/Bridge/PhpUnit/ClockMock.php]: Indeed, this is weird :thinking: Reverted this one as well as your second comment

**Review by derrabus — APPROVED** [review]

**Comment by derrabus** [comment]
Thank you @alexandre-daubois.

File: src/Symfony/Component/Runtime/Internal/ComposerPlugin.php [hunk 3]
```diff
@@ -83,7 +83,7 @@ public function updateAutoloadFile(): void
         }
 
         if (!$nestingLevel) {
-            $projectDir = '__'.'DIR__.'.var_export('/'.$projectDir, true);
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Runtime/Internal/ComposerPlugin.php:86]: FYI this was on purpose and breaks the plugin
@lyrixx is on a PR to revert this change and add a comment on top :)

Comment (alexandre-daubois) [reply src/Symfony/Component/Runtime/Internal/ComposerPlugin.php:86]: Oh right, thanks for letting me know!
